tests/qubo_tools/test_device.py

```
@@ -0,0 +1,159 @@
+
+import pytest
+
+from mqt.problemsolver.qubo_tools import Calibration
+
+
+@pytest.fixture
+def sample_device_heavy_hex() -> Calibration:
+    """Provides a test device calibration with heavy-hex-style connectivity."""
+    heavy_hex_coupling = []
+
+    def get_qubit_index(row: int, cell: int, off: bool) -> int:
```

Ruff (0.14.1)

Boolean-typed positional argument in function definition

(FBT001)